python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -1024,6 +1077,168 @@ def _schedule_row_to_schedule(cls,
             self._schedule_first_task(schedule_row, now)
             self._resume_check_schedules()
 
+    async def disable_schedule(self, schedule_id):
+        """
+        Find running Schedule, Terminate running process, Disable Schedule, Update database
+
+        :param schedule_id:
+        :return:
```

Comment:
Please follow one style for docstring, as we have mentioned in **__template__.py**. Like below

Args:
Returns: